src/Lykke.Service.GoogleAnalyticsWrapper.Core/Domain/Health/HealthIssuesCollection.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Lykke.Service.GoogleAnalyticsWrapper.Core.Domain.Health
+{
+    public class HealthIssuesCollection : IReadOnlyCollection<HealthIssue>
```

Comment:
What for this class? Any List<> can be casted to IReadOnlyCollection

Reply:
its from the service template